events/2024-06-10-mtb-ngs.html

```diff
@@ -51,8 +51,6 @@
     channel: event-mtb-ngs
 
 program:
-  - section: "Day 1: Overview of NGS technologies & TB specific NGS solutions"
-    description: |
```

Comment:
so you would like this text to appear above the "Day 1" header right? (we will have to do that a bit differently, but we can make that work)